### ISSUE TITLE
Please clarify license

### ISSUE THREAD

**Issue by hajs** [larsac07/Motion-Highcharts-Plugin#17]
Hello!

I am not sure if I can commercially use this plugin because

`package.json` contains the following entry:
```json
  "license": "CC-BY-NC-SA-4.0",
```

But the file `motion.js` contains the following line:
```javascript
@license http://creativecommons.org/licenses/by-sa/4.0/ Creative Commons Attribution-ShareAlike 4.0 International (CC BY-SA 4.0)
```

The offical plugin registry (https://www.highcharts.com/products/plugin-registry/single/40/Motion) also claims that the license is CC BY-SA 4.0.

**Comment by larsac07** [comment]
Closed by c8c2ccdbca3d6e2fe1d8ba9d715966d31634a32d

**Comment by larsac07** [comment]
And thank you for reporting it :)
